### PR TITLE
add detail from beast page and make a few style tweaks

### PR DESCRIPTION
**Pull request by EmmaFrith**
* make EYTS lower case, in line with QTS
* add detail about EYITT (from beast page)
* make 'assessment only' lower case in line with Ways to train page
* remove repeated link to accredited providers
* remove repetition from intro

### CODE REVIEW

File: app/views/content/early-years-teaching-training.md [hunk 1]
```diff
@@ -55,33 +55,35 @@ calls_to_action:
...
 Your training will enable you to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
- 
+
 To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
 
-* Maths grade 4 (C)
+* maths grade 4 (C)
 * English grade 4 (C)
-* Science grade 4 (C)
+* science grade 4 (C)
```
Comment (peteryates) [inline app/views/content/early-years-teaching-training.md:80]: Aren't subject names proper nouns?

Comment (MylesJarvis) [reply app/views/content/early-years-teaching-training.md:80]: image

I believe it's only English

File: app/views/content/early-years-teaching-training.md [hunk 2]
```diff
@@ -55,33 +55,35 @@ calls_to_action:
         Salary: "No"
         Funding available: "No"
         Length of training: "3 months"
-        Qualification: "Early Years Teacher Status"
-      - "Graduate Assessment-Only training"
+        Qualification: "Early years teacher status"
+      - "Graduate 'assessment only' training"
```
[the remark this answers is not in the document]
Comment (MylesJarvis) [reply app/views/content/early-years-teaching-training.md:59]: We've used it without hyphen elsewhere in the site, including the AO provider page.

File: app/views/content/early-years-teaching-training.md (hunk 1 above)
Comment (EmmaFrith) [reply app/views/content/early-years-teaching-training.md:80]: on Apply for teacher training we user lower case for everything except English, and on gov.uk you'll see a lot of lower case going on e.g. https://www.gov.uk/government/publications/gcse-single-science

File: app/views/content/early-years-teaching-training.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (EmmaFrith) [reply app/views/content/early-years-teaching-training.md:59]: i've stuck to what's on gov.uk, and what we have elsewhere on the site. technically it should probably have a hyphen but maybe one to think about seperately?

File: app/views/content/early-years-teaching-training.md [hunk 3]
```diff
@@ -91,25 +93,19 @@ A full-time bachelor’s degree in an early childhood-related subject leading to
...
-This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards). 
+This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
 
 Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
 
-$EY-self-funded$ 
-
-This self-funded route is ideal for graduates with early-years experience. If you already meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards), you can gain EYTS without the need for further training. 
-
-## Next Steps
```
Comment (MylesJarvis) [inline app/views/content/early-years-teaching-training.md:111]: What's the logic for removing a next steps section?

Comment (EmmaFrith) [reply app/views/content/early-years-teaching-training.md:111]: there were just two links in this section - one link repeated a link further up the page, so I just deleted that one and move the other one up too

Comment (MylesJarvis) [reply app/views/content/early-years-teaching-training.md:111]: Cool - I think this can be revisited as part of our CTA review for these 'offramp' pages.

**Review by MylesJarvis — APPROVED** [review]

**Pull request merged**